itests/org.jboss.tools.quarkus.integration.tests/src/org/jboss/tools/quarkus/integration/tests/project/CreateNewQuarkusConfigurationGradleTest.java

```diff
@@ -0,0 +1,121 @@
+import org.eclipse.reddeer.eclipse.ui.console.ConsoleView;
+import org.eclipse.reddeer.eclipse.ui.navigator.resources.ProjectExplorer;
+import org.eclipse.reddeer.junit.runner.RedDeerSuite;
+import org.eclipse.reddeer.requirements.openperspective.OpenPerspectiveRequirement.OpenPerspective;
+import org.eclipse.reddeer.swt.condition.ShellIsAvailable;
+import org.eclipse.reddeer.swt.impl.button.PushButton;
+import org.eclipse.reddeer.swt.impl.menu.ContextMenuItem;
+import org.eclipse.reddeer.swt.impl.table.DefaultTableItem;
+import org.eclipse.reddeer.swt.impl.tree.DefaultTreeItem;
+import org.eclipse.reddeer.workbench.impl.shell.WorkbenchShell;
+import org.jboss.tools.quarkus.reddeer.perspective.QuarkusPerspective;
+import org.jboss.tools.quarkus.reddeer.wizard.CodeProjectTypeWizardPage;
+import org.jboss.tools.quarkus.reddeer.wizard.QuarkusWizard;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * https://issues.redhat.com/browse/JBIDE-27065
+ * New quarkus launch configuration integration test for Gradle
+ * @author olkornii@redhat.com
+ */
+@OpenPerspective(QuarkusPerspective.class)
+@RunWith(RedDeerSuite.class)
+public class CreateNewQuarkusConfigurationGradleTest extends LaunchConfiguration {
+
+	public CreateNewQuarkusConfigurationGradleTest() {
+		super("Create New Quarkus Configuration Gradle Test");
+	}
+
+	private static String GRADLE_PROJECT_NAME = "code-with-quarkus"; // w8ing for fix https://issues.redhat.com/browse/JBIDE-27057
+	private static ConsoleView consoleView;
+
+	@BeforeClass
+	public static void testNewNewQuarkusGradleProject() {
+		new WorkbenchShell().setFocus();
+
+		QuarkusWizard qw = new QuarkusWizard();
+		qw.open();
+		assertTrue(qw.isOpen());
+
+		CodeProjectTypeWizardPage wp = new CodeProjectTypeWizardPage(qw);
+		wp.setProjectName(GRADLE_PROJECT_NAME);
+		wp.setGradleProjectType();
+
+		qw.next();
+		qw.finish(TimePeriod.VERY_LONG);
+
+		assertTrue(new ProjectExplorer().containsProject(GRADLE_PROJECT_NAME));
+
+		consoleView = new ConsoleView();
+	}
+
+	@Test
+	public void testNewQuarkusGradleConfiguration() {
+		CreateNewQuarkusGradleConfiguration();
+		CheckNewQuarkusGradleConfiguration();
+		RunNewQuarkusGradleConfiguration();
+	}
+
+	public void CreateNewQuarkusGradleConfiguration() {
+		new WorkbenchShell().setFocus();
+		new ProjectExplorer().selectProjects(GRADLE_PROJECT_NAME);
+
+		new ContextMenuItem("Run As", "Run Configurations...").select();
+		new WaitUntil(new ShellIsAvailable("Run Configurations"));
+
+		new DefaultTreeItem("Quarkus Application").select();
+		new ContextMenuItem("New Configuration").select();
+
+		setName(GRADLE_PROJECT_NAME + "Configuration");
+		
+		new PushButton("Browse...").click();
+		new DefaultTableItem(GRADLE_PROJECT_NAME).select();
+		new PushButton("OK").click();
+
+		new PushButton("Apply").click();
+		new PushButton("Close").click();
+	}
+
+	public void CheckNewQuarkusGradleConfiguration() {
```

Comment:
Also similar with CheckNewQuarkusMavenConfiguration, you can put it in Abstract class with parameter for project name.